<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewHabitWindow</class>
 <widget class="QWidget" name="NewHabitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>119</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>119</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Новая привычка</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>114</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Название привычки:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>46</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Частота:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>114</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Кол-во для стрика:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Каждую секунду (тест)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ежедневно</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Раз в неделю</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Раз в месяц</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Добавить привычку</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Вернутся</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
